Makefile.include

```diff
@@ -213,7 +216,7 @@ all: ..compiler-check ..build-message $(USEPKG:%=${BINDIR}%.a) $(APPDEPS)
 	$(AD)DIRS="$(DIRS)" "$(MAKE)" -C $(CURDIR) -f $(RIOTBASE)/Makefile.application
 ifeq (,$(RIOTNOLINK))
 ifeq ($(BUILDOSXNATIVE),1)
-	$(AD)$(if $(CPPMIX),$(CXX),$(LINK)) $(UNDEF) -o $(ELFFILE) $(BASELIBS) $(LINKFLAGS) $(LINKFLAGPREFIX)-no_pie
+	$(AD)$(if $(CPPMIX),$(CXX),$(LINK)) $(UNDEF) -o $(ELFFILE) $(filter-out $(EMPTYARCHIVES), $(BASELIBS)) $(LINKFLAGS) $(LINKFLAGPREFIX)-no_pie
```

Comment:
This leaves me with a linker command without any archives when I call it with `make clean all`. The archives cannot be found by `find` as they are removed before. Issuing a consecutive `make all` succeeds then.